cylc/flow/scripts/cylc.py

```diff
@@ -504,7 +498,7 @@ def list_plugins():
 
 
 @contextmanager
-def pycoverage(cmd_args):
+def pycoverage(cmd_args):  # pragma: no cover
```

Comment:
This stops codecov complaining about this method (the one that runs the coverage itself) not being covered.